### ISSUE TITLE
Mutual Information vs Normalized mutual info score

### ISSUE THREAD

**Issue by jihyeonseong** [connorlee77/pytorch-mutual-information#7]
Hello, I have some questions regarding your implementation.

It is an awesome implementation considering multivariate data.
And I'm trying to implement on time series dataset.
Here is my question below.

1. Can this MI score value be different from sklearn normalized mutual info score?
2. Can this MI score value be negative?

Thank you in advance.
